files/image_config/timezone/timezone-config.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+CURRENT_TIMEZONE=`cat /etc/timezone`
+TIMEZONE=`sonic-cfggen -d -v DEVICE_METADATA[\'localhost\'][\'timezone\']`
+
+if [ ! $TIMEZONE == "" ]
```

Comment:
Suggest `if [ -n $TIMEZONE ]` to test for non-zero length.

Reply:
modified as you suggested

Reply:
Please also add double-quotes around `$TIMEZONE` as I suggested.

Reply:
> Please also add double-quotes around `$TIMEZONE` as I suggested.
done